### ISSUE TITLE
Node visibility can't be set

### ISSUE THREAD

**Issue by nicolaeschbach** [cyber-geiger/toolbox-storage#32]
**Describe the bug**
Node visibility is always red. It doesn't take the constructor visibility argument nor change its state after change the visibility field.

**Environmental information**
- geiger_localstorage 0.6.15

**To Reproduce**
```dart
    NodeImpl node1 = NodeImpl("NodeTLPBlack", "testOwner", ":Enterprise", Visibility.black);
    print(node1.visibility);
    node1.visibility = Visibility.green;
    print(node1.visibility);
```
**Expected behavior**
Node visibility corresponds to the set value.

**Comment by martingwerder** [comment]
This should be addressed in the most recent relase. Can you confirm?

**Comment by nicolaeschbach** [comment]
Confirm. This issue is fixed with the 0.6.17 release.
